ui/Button.py

```diff
@@ -0,0 +1,48 @@
+import pygame
+
+class Button(): 
+    def __init__(self, default_color, hover_color, width, height, x, y, action, image, x_image, y_image, text, x_text, y_text):
```

Comment:
Je me dis maintenant que pour faire un peu moins de paramètres à entrer, on pourait mettre les coordonnées et la taille dans un rect pygame nan ? Et les coordonées dans des tuples ?
```suggestion
    def __init__(self, default_color, hover_color, buttonRect, action, image, imageCoordinates, text, textCoordinates):
```